app/views/state_file/questions/id_grocery_credit/edit.html.erb

```diff
@@ -33,14 +33,14 @@
             <div class="white-group">
               <p class="spacing-below-15"><strong><%= t(".select_household_members", tax_year: current_tax_year) %></strong></p>
               <div class="tight-checkboxes spacing-above-0 spacing-below-0">
-                <%= f.cfa_checkbox(:primary_has_grocery_credit_ineligible_months, current_intake.primary.full_name, options: { checked_value: "yes", unchecked_value: "no", "data-follow-up": "#primary_months_ineligible_for_grocery_credit" }) %>
                 <% if current_intake.filing_status_mfj? %>
                   <%= f.cfa_checkbox(:spouse_has_grocery_credit_ineligible_months, current_intake.spouse.full_name, options: { checked_value: "yes", unchecked_value: "no", "data-follow-up": "#spouse_months_ineligible_for_grocery_credit" }) %>
                 <% end %>
                 <%= f.fields_for :dependents do |ff| %>
                   <% dependent = ff.object %>
                   <%= ff.cfa_checkbox(:id_has_grocery_credit_ineligible_months, dependent.full_name, options: { checked_value: "yes", unchecked_value: "no", "data-follow-up": "#dependent_#{dependent.id}_months_ineligible_for_grocery_credit" }) %>
                 <% end %>
+                <%= f.cfa_checkbox(:primary_has_grocery_credit_ineligible_months, current_intake.primary.full_name, options: { checked_value: "yes", unchecked_value: "no", "data-follow-up": "#primary_months_ineligible_for_grocery_credit" }) %>
```

Comment:
I moved this so that the error could be attached to primary and render at the bottom of the list of family members